django/db/migrations/autodetector.py

```diff
@@ -521,6 +522,9 @@ def generate_created_models(self):
                     related_fields[field.name] = field
                 if getattr(field.remote_field, "through", None) and not field.remote_field.through._meta.auto_created:
                     related_fields[field.name] = field
+            # Are there any indexes to defer?
+            indexes = model_state.options['indexes']
```

Comment:
`.pop()` deletes the key from the dictionary. Unlike other options below, we don't want to delete the key (we want to empty the list).